### PR TITLE
Tools: Rename indicator types to camel case.

### PR DESCRIPTION
**Pull request by activist**
Ref. internal discussion

### CODE REVIEW

File: ts/indicators/acceleration-bands.src.ts [hunk 1]
```diff
@@ -16,40 +16,40 @@ import H from '../parts/Globals.js';
...
 
-        class AbandsIndicator
-            extends SmaIndicator implements MultipleLinesIndicator {
-            public options: AbandsIndicatorOptions;
-            public data: Array<AbandsIndicatorPoint>;
+        class ABandsIndicator
+            extends SMAIndicator implements MultipleLinesIndicator {
+            public options: ABandsIndicatorOptions;
+            public data: Array<ABandsIndicatorPoint>;
             public linesApiNames: MultipleLinesMixin['linesApiNames'];
```
Comment (bre1470) [inline ts/indicators/acceleration-bands.src.ts]: Not related to naming, but this one is lacking `pointClass` and `points` properties.

File: ts/indicators/aroon.src.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ import H from '../parts/Globals.js';
 declare global {
     namespace Highcharts {
         class AroonIndicator
-            extends SmaIndicator implements MultipleLinesIndicator {
+            extends SMAIndicator implements MultipleLinesIndicator {
             public data: Array<AroonIndicatorPoint>;
             public nameBase: string;
             public linesApiNames: MultipleLinesMixin['linesApiNames'];
```
Comment (bre1470) [inline ts/indicators/aroon.src.ts]: This one lacks `pointClass` and `points` properties too.

File: ts/indicators/atr.src.ts [hunk 3]
```diff
@@ -16,29 +16,29 @@ import H from '../parts/Globals.js';
  */
 declare global {
     namespace Highcharts {
-        class AtrIndicator extends SmaIndicator {
-            public data: Array<AtrIndicatorPoint>
+        class ATRIndicator extends SMAIndicator {
+            public data: Array<ATRIndicatorPoint>
             public yData: Array<Array<number>>;
             public getValues(
```
Comment (bre1470) [inline ts/indicators/atr.src.ts]: The same here.

File: ts/indicators/atr.src.ts [hunk 4]
```diff
@@ -16,29 +16,29 @@ import H from '../parts/Globals.js';
...
         }
 
-        interface AtrIndicatorOptions extends SmaIndicatorOptions {
-            params?: AtrInidicatorParamsOptions;
+        interface ATRIndicatorOptions extends SMAIndicatorOptions {
+            params?: ATRInidicatorParamsOptions;
         }
 
-        interface AtrInidicatorParamsOptions {
+        interface ATRInidicatorParamsOptions {
```
Comment (bre1470) [inline ts/indicators/atr.src.ts]: Typo: `…Inidicator…`

File: ts/indicators/acceleration-bands.src.ts (hunk 1 above)
Comment (activist) [reply ts/indicators/acceleration-bands.src.ts]: Yes, there are a few things that I have on my list. Also some missing inheritance in another indicator. Just fix it in this PR or create another?

Comment (bre1470) [reply ts/indicators/acceleration-bands.src.ts]: As long as it goes into a separate commit, this PR can be used.

**Comment by activist** [comment]
@bre1470 Please review, I have added the requested changes and then some :)

File: ts/indicators/wma.src.ts [hunk 5]
```diff
@@ -64,21 +62,16 @@ var seriesType = H.seriesType;
...
     i: number,
     index: number
 ): void {
-    var xValue = xVal[i],
-        yValue = index < 0 ? yVal[i] : (yVal[i] as any)[index];
+    var xValue: number = xVal[i],
+        yValue: (number|Array<number>) = index < 0 ? yVal[i] : yVal[i][index];
 
-    points.push([xValue, yValue]);
+    points.push([xValue, (yValue as any)]);
```
Comment (bre1470) [inline ts/indicators/wma.src.ts]: I have the impression that this is not correct. Actually points can be `Array<[number, (number|Array<number>)]>`, if the yValues can be either `number` or `Array<number>`

File: ts/indicators/wma.src.ts [hunk 6]
```diff
@@ -189,8 +175,8 @@ seriesType<Highcharts.WmaIndicator>(
 
             // Switch index for OHLC / Candlestick
             if (isArray(yVal[0])) {
-                index = (params.index);
-                yValue = (yVal[0] as any)[index];
+                index = (params.index as any);
```
Comment (bre1470) [inline ts/indicators/wma.src.ts]: Maybe `params.index` needs to be non-optional defined.

File: ts/indicators/wma.src.ts (hunk 5 above)
Comment (activist) [reply ts/indicators/wma.src.ts]: Thanks, that was nicely spotted. I totally missed that one.

File: ts/indicators/wma.src.ts (hunk 6 above)
Comment (activist) [reply ts/indicators/wma.src.ts]: Changed to be non-optional.

**Comment by activist** [comment]
@bre1470 Please review added changes.

**Pull request merged**